File: fixtures/node_output/get_with_single_header.js
var request = require('request');

var headers = {
    'foo': 'bar'
};

var options = {
    url: 'http://example.com/',
    headers: headers
};

function callback(error, response, body) {
    if (!error && response.statusCode == 200) {
        console.log(body);
    }
}

request(options, callback);

File: fixtures/elixir_output/get_with_single_header.ex
method = :get
url = "http://example.com/"
headers = [
  {~s|foo|, ~s|bar|},
]
body = nil
options = []
params = []

request = %HTTPoison.Request{
  method: method,
  url: url,
  body: body,
  headers: headers,
  options: options,
  params: params,
}

response = HTTPoison.request(request)
